File: fizzBuzz.py
def fizzBuzz():
    output = ""
    for num in range(1, 101):
        fzbz = True
        if num %3 == 0:
            output = output + "Fizz"
            fzbz = False
        if num %5 == 0:
            output = output + "Buzz"
            fzbz = False
        if fzbz:
            output = output + str(num)
        if num != 100:
            output = output + ","
    print(output)

fizzBuzz()

File: src/main/java/com/github/raphamendonca/raphamendoncafizzbuzz/FizzBuzz.java
package com.github.raphamendonca.raphamendoncafizzbuzz;

import io.swagger.annotations.ApiModel;

@ApiModel
public class FizzBuzz {

	private int number;

	public FizzBuzz(Integer number) {
		this.number = number;
	}

	public int getNumber() {
		return number;
	}

	public String getValue() {
		StringBuilder output = new StringBuilder();
		
		Boolean isFizzBuzzed = Boolean.TRUE;
		if (number % 3 == 0) {
			output.append("Fizz");
			isFizzBuzzed = Boolean.FALSE;
		}
		if (number % 5 == 0) {
			output.append("Buzz");
			isFizzBuzzed = Boolean.FALSE;
		}
		if (Boolean.TRUE.equals(isFizzBuzzed)) {
			output.append(number);
		}
		
		return output.toString();
	}
	
}
